lib/p2p/Pool.ts

```diff
@@ -357,7 +358,10 @@ class Pool extends EventEmitter {
         break;
       }
       case PacketType.GetOrders: {
-        this.emit('packet.getOrders', peer, packet.header.id);
+        const getOrdersPacketBody = (packet as packets.GetOrdersPacket).body!;
+        const pairIds = getOrdersPacketBody ? getOrdersPacketBody.pairIds : [];
+        this.logger.verbose(`sending orders to ${peer.nodePubKey} for supported pairs: ${pairIds}`);
```

Comment:
we're not sending orders from here, but merely emitting an event. You can log it as `get_orders` being received, or drop the logging.

Reply:
right, would it be sufficient to change the log as "got orders" instead of "sending orders" ?

Reply:
The would be quite confusing, because you didn't received orders, but rather orders request. I'd suggest to drop the log altogether. We are logging the wire protocol anyway.

Reply:
Alright, thanks for the explanation